homework/shelkovich_ilya/product/src/main/java/com/tutrit/java/quickstart/service/ProductService.java

```diff
@@ -0,0 +1,20 @@
+package com.tutrit.java.quickstart.service;
+
+import com.tutrit.java.quickstart.bean.Product;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ProductService {
+
+    Logger log = LoggerFactory.getLogger("ProductService");
+
+    public double actualPrice(double regularPrice, double discount) {
```

Comment:
Well, you working with product - not with regular price. You named your service ProductService. Consider to use Product as a parameter. And use its, products, regular price.

Reply:
made Product as a parameter and refactored method